easybuild/easyconfigs/s/SOLAR-Eclipse/SOLAR-Eclipse-8.1.1.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Tarball"
+
+name = 'SOLAR-Eclipse'
+version = '8.1.1'
+
+homepage = 'http://solar-eclipse-genetics.org/'
+description = """SOLAR-Eclipse is an extensive, flexible 
+software package for genetic variance components analysis, 
+including linkage analysis, quantitative genetic analysis, 
+SNP association analysis (QTN and QTLD), and covariate screening. """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://www.nitrc.org/frs/download.php/8542/']
+sources = ['solar_linux.tar.gz']
+
+checksums = ['cbc05fb6c39853a888d9ee151a383461']
+
+postinstallcmds = ["./install_solar %(installdir)s %(installdir)s",
+               "./install_rlwrap %(installdir)s"]
```

Comment:
@rtripath89 please also use `install_cmd` rather than `postinstallcmds` (which was the intention of switching to `PackedBinary`)